quicksort/quicksort.py

```diff
@@ -0,0 +1,16 @@
+def s(l,st,end):
+    print("st,end: ", st, end)
+    if len(l) <= 1: return
+    idx = random.randint(st,end)
+    l[idx],l[st]=l[st],l[idx]
+    pivot=l[st]
+    small=st
+    big = st
```

Comment:
It is confusing that you initialize big with a value here. It will be initialized in the for loop below I believe so it is not necessary to initialize big.

Reply:
I have tried it without it - same issue is in loop condition - because my dataset is l = [1,4,2,6,10,6,5]